Dockerfile

```diff
@@ -183,5 +204,9 @@ RUN \
 # 
 # Copy shell scripts and config files over
 COPY bin/* /usr/local/bin/
+COPY python/requirements*.txt ./
+RUN pip install --upgrade pip ;\
+    pip install -r requirements-pre.txt; \
```

Comment:
Please split into multiple `RUN` commands, to make dockerfile build cache effective

Reply:
On second thought: Did we not want to only install `gdal` python package?
Maybe simply do
```
RUN pip install gdal....
```
directly without a requirements.txt usage

Reply:
I'm installing rasterio as well, so I will keep requirements.txt

Reply:
Why also rasterio now?